bitbucket-pipelines.dist.yml

```diff
@@ -0,0 +1,82 @@
+
+definitions:
+  setupdeps: &setupDeps
+    export DEBIAN_FRONTEND='noninteractive' ;
+    export COMPOSER_ALLOW_SUPERUSER=1 ;
+    export COMPOSER_CACHE_DIR='/root/.composer/cache' ;
+    export MOODLE_START_BEHAT_SERVERS='NO' ;
+    export CI_BUILD_DIR='/tmp/plugin' ;
+    export MOODLE_BEHAT_WWWROOT="http://$BITBUCKET_DOCKER_HOST_INTERNAL:8000" ;
+    export MOODLE_BEHAT_WDHOST="http://$BITBUCKET_DOCKER_HOST_INTERNAL:4444/wd/hub" ;
+    export MOODLE_REPO='https://bitbucket.org/moodle/moodle.git' ;
+    export MOODLE_BRANCH='MOODLE_311_STABLE' ;
```

Comment:
Is this going to cause a problem for the Moodle 4.0 test? Or should it be moved to the Moodle 3.11 test?

Reply:
These are only the defaults. If nothing specific is set in moodle-plugin-ci install these are the default values used. However, anything specified in moodle-plugin-ci install command line overrides any ENV variable.